src/comments.ts

```diff
@@ -78,12 +78,12 @@ and I'll merge this PR almost instantly. Thanks for helping out! :heart:
 ${otherOwners.length === 0 ? "" : `
 (${otherOwners.map(o => "@" + o).join(", ")}: you can do this too.)`}`}));
 
-export const WaitUntilMergeIsOK = (user: string, abbrOid: string, uri: string) => ({
+export const WaitUntilMergeIsOK = (user: string, abbrOid: string, uri: string, mainCommentID: number | undefined) => ({
     // at most one reminder per update
     tag: `wait-for-merge-offer-${abbrOid}`,
     status: `:passport_control: Hi @${user},
 
-I can't [accept a merge request](${uri}) until the PR has a green CI and was appropriately reviewed. I will let you know once that happens.
+I can't [accept a pull request](${uri}) until all of the checks in the "Status" section of [this comment](#issuecomment-${mainCommentID || ""}) are green. I will let you know once that happens.
```

Comment:
the undefined should never really happen (because the bot should always have commented before this message would show, but better to not have an unpredictable `undefined` IMO)

Reply:
Maybe use `"???"` instead?  Just in case something breaks, it would look more obvious like a "something unexpected happened".